Code/Framework/AzCore/AzCore/IO/Streamer/StreamerContext.h

```diff
@@ -7,23 +7,28 @@
  */
 #pragma once
 
-#include <AzCore/base.h>
-#include <AzCore/IO/Streamer/FileRequest.h>
 #include <AzCore/IO/Streamer/Statistics.h>
 #include <AzCore/IO/Streamer/StreamerConfiguration.h>
 #include <AzCore/IO/Streamer/StreamerContext_Platform.h>
+#include <AzCore/Statistics/RunningStatistic.h>
+#include <AzCore/base.h>
```

Comment:
There is a reason `AzCore/base.h` is the first include and that is to make sure that any includes of base level platform macros/defines are set

Reply:
Also keeps headers included in alphabetic order that way.

Reply:
clang format sorted those ( it sorts case-sensitively ), will fix

Reply:
Is there a way to fix the .clang-format file to sort in a case insensitive matter so this doesn't happen again? It looks like we can now set the option `SortIncludes` to be `CaseInsensitive` instead of just `true` - https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
I believe it's clang13 option :frowning_face:

Reply:
Aaargh you're right 😩 I just checked and VS2019 just as clang-12 (the default we use)

```
λ "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Tools\Llvm\x64\bin\clang-format.exe" --version
clang-format version 12.0.0
```

Hmm hopefully we can update to VS2022 before too long and that should hopefully have bumped clang-format versions (we did at one stage consider building our own version we could distribute with the repo so we could always get the latest and greatest changes but there were enough downsides to now make it worth it unfortunately 🙁 One thing to do with this manual fix is to add `// clang-format off` and `// clang-format on` around it to stop it changing if other people format the file in future, and when we move to clang-format 13 we can then delete these lines.

Thanks for checking 👍